art/settings/base.py

```diff
@@ -37,15 +37,17 @@
 SECRET_KEY = config('SECRET_KEY')
 DEBUG = config('DEBUG', default=False, cast=bool)
 DATABASES = {
-    'default': dj_database_url.config()
+    'default': dj_database_url.config(
+        default=config('DATABASE_URL')
+    )
 }
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
 ALLOWED_HOSTS = [
```

Comment:
this should be env specific - `art-backend.herokuapp.com` for prod and `'127.0.0.1',  '0.0.0.0',` for dev

Reply:
Should I have them removed from `base.py` when I place them in the other envs?

Reply:
yea.. the values in prod or dev envs should not be in base